src/Kanbas/Courses/Quizzes/List/index.tsx

```diff
@@ -1,22 +1,145 @@
+    navigate(`/Kanbas/Courses/${courseId}/Quizzes/${quizId}`);
+  };
+
+  const handlePublishToggle = (quizId: string) => {
+    const _isPublished =
+      quizzesList.find((q) => q._id == quizId)?.isPublished ?? false;
+
+    postQuizSetPublish(quizId, !_isPublished).then(() => {
+      dispatch(toggleQuizPublished(quizId));
+    });
+  };
 
-    return (
+  const getDateDisplay = (date: Date) => {
+    return "";
+  };
+
+  const getAvailableDateDisplay = (
+    availableFromDate: Date,
+    availableUntilDate: Date
+  ) => {
+    const today = new Date();
+
+    if (today < availableFromDate) {
+      return (
         <>
-            {/* TODO - update this to pass the quiz id to this function for navigation */}
-            <a onClick={() => handleQuizSelection()} />
+          <b>Not Available until</b> {availableFromDate.toDateString()}{" "}
         </>
-    );
+      );
+    }
+
+    if (today < availableUntilDate) {
+      return <b>Available</b>;
+    }
+
+    return <b>Closed</b>;
+  };
+
+  useEffect(() => {
+    if (courseId !== undefined) {
+      getCourseQuizzes(courseId).then((quizzes: IKanbasQuiz[]) => {
+        dispatch(setQuizzes(quizzes));
+      });
+    }
+  }, [dispatch]);
+
+  return (
+    <>
+      <ul className="list-group wd-quizzes-grid">
+        <li className="list-group-item">
+          <div
+            className="wd-quizzes-grid-section d-flex align-items-center"
+            onClick={() => setQuizzesExpanded(!quizzesExpanded)}
+            aria-controls={"quiz-section-collapse"}
+            aria-expanded={quizzesExpanded}
+          >
+            <FaCaretDown
+              style={{ transform: quizzesExpanded ? "" : "rotate(-90deg)" }}
+            />
+            <div className="ms-1 me-auto">Assignment Quizzes</div>
+          </div>
+
+          <Collapse in={quizzesExpanded}>
+            <div
+              id={"quiz-section-collapse"}
+              className="wd-quiz-section-collapse"
+            >
+              <ul className="list-group">
+                {quizzesList?.map((q, qIndex) => (
+                  <li
+                    key={qIndex}
+                    className="list-group-item d-flex align-items-center"
+                  >
+                    <FaRocket className="ms-2 wd-icon-green" />
+                    <div className="wd-quizzes-grid-content-text flex-fill me-auto">
+                      <a
+                        onClick={() => handleQuizSelection(q._id)}
+                        className="wd-event-link"
+                      >
+                        {q.title}
+                      </a>
+                      <p className="wd-quizzes-grid-content-details">
+                        {getAvailableDateDisplay(
+                          q.availableDate,
+                          q.availableUntilDate
+                        )}{" "}
+                        | Due {q.dueDate.toString()} | {q.points} Points |{" "}
+                        {q.questionsCount} Questions
```

Comment:
Missing logic for handling isMultipleDatesAvailable?

Reply:
I don't think we're required to support it. I'll confirm in prof's OH.

Reply:
Leaving it for later so we can get this PR in.